File: CS/Q144847/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Xpo;
using DevExpress.Xpo.DB;

namespace Q144847 {
    public partial class Form1 : Form {
        public Form1() {
            XpoDefault.ConnectionString = MSSqlConnectionProvider.GetConnectionString("(local)", "Northwind");
            InitializeComponent();
            UpdateLabel();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            if (xpPageSelector1.CurrentPage == 0) return;
            xpPageSelector1.CurrentPage--;
            UpdateLabel();
        }

        private void UpdateLabel() {
            labelControl1.Text = string.Format("Page {0} of {1}", xpPageSelector1.CurrentPage,
                xpPageSelector1.PageCount);
        }

        private void simpleButton2_Click(object sender, EventArgs e) {
            if (xpPageSelector1.CurrentPage == xpPageSelector1.PageCount - 1) return;
            xpPageSelector1.CurrentPage++;
            UpdateLabel();
        }
    }
}

File: VB/Q144847/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.Xpo
Imports DevExpress.Xpo.DB

Namespace Q144847

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            XpoDefault.ConnectionString = MSSqlConnectionProvider.GetConnectionString("(local)", "Northwind")
            InitializeComponent()
            UpdateLabel()
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            If xpPageSelector1.CurrentPage = 0 Then Return
            xpPageSelector1.CurrentPage -= 1
            UpdateLabel()
        End Sub

        Private Sub UpdateLabel()
            labelControl1.Text = String.Format("Page {0} of {1}", xpPageSelector1.CurrentPage, xpPageSelector1.PageCount)
        End Sub

        Private Sub simpleButton2_Click(ByVal sender As Object, ByVal e As EventArgs)
            If xpPageSelector1.CurrentPage = xpPageSelector1.PageCount - 1 Then Return
            xpPageSelector1.CurrentPage += 1
            UpdateLabel()
        End Sub
    End Class
End Namespace
